SmartAPIRegistration.md

```diff
@@ -0,0 +1,51 @@
+# SmartAPI Registrations
+
+
+## Background
+
+The SmartAPI registry is Translator's discovery registry.  Translator tools create a yaml registration that can be queried through the SmartAPI registry. 
+
+This document defines the Translator requirements for such registrations.
+
+Translator components are registered using OpenAPI metadata as described [here](https://smart-api.info/guide).   In addition to those general requirements, specific additions are required for Translator components.
+
+## Requirements
+
+### Translator tag
+
+"translator" must be included as a top-level OpenAPI tag.
+
+### x-translator element
+
+The registration must contain a compliant x-translator element as described [here](https://github.com/NCATSTranslator/translator_extensions/).  Among other things, the x-translator element defines the infores identifier for this component: an identifier that is common across environments such as dev, test, and production.
```

Comment:
can use this more specific link for `x-translator`:

https://github.com/NCATSTranslator/translator_extensions/tree/production/x-translator